doc/source/features/online-schema-changes.rst

```diff
@@ -0,0 +1,82 @@
+        
+        All other writes must wait until the DDL statement is executed. While a DDL statement is running, any long-running transactions in progress and using the same resource receive a deadlock error at commit and are rolled back. There is no way to verify if individual nodes have succeeded in processing the DDL statement. 
+    * - `RSU`
+      - Rolling  Schema Upgrade
+
+        A DDL statement are processed on each nod locally. This action guarantees high availability. The node desynchronizes with the cluster and disables flow control while executing the DDL statement. The rest of the cluster is not affected. After the statement execution, the node applies delayed events and synchronizes with the cluster. 
+
+        The cluster continues to operate, although during the process, some nodes may be updated and some nodes may have the older schema structure.
+
+        For this method, the ``gcache`` must be large enough to store the data for the duration of the DDL change. 
+    * - `NBO`
+      - Non-Blocking Operation
+
+        This method is similar to ``TOI``. All nodes acquire metadata locks on each table or schema for any open transactions and do not release those locks until the transaction ends. The DDL statements execute on all nodes after the transactions have completed. This method avoids the TOI issue when a long-running DDL blocks other updates in the cluster.
+
+        For this method, the ``gcache`` must be large enough to store the data for duration of the DDL statement. 
+
+Using the NBO method
+----------------------
+
+You set the method for online schema changes by either using the ``wsrep_OSU_method`` variable in ``my.cnf`` or  using the ``SET`` statement in a MySQL client. 
+
+To set the variable in the my.cnf file, do the following:
+
+.. sourcecode:: text
+
+    wsrep_osu_method=NBO
+
+Restart the server. 
+
+To dynamically set the ``NBO`` method in the client, run the following statement:
+
+.. sourcecode:: mysql
+
+    SET GLOBAL wsrep_OSU_method='NBO';
+
+
+Limitations
+-----------------
+
+The following items are the limitations of using ``NBO``:
+
+* Write attempts are blocked until the ``ALTER`` statement completes. If the DDL statement is ``ALTER TABLE ... LOCK=SHARED``, read operations are allowed. If the statement is ``ALTER TABLE ... LOCK=EXCLUSIVE``, read operations are not allowed.
+
+* Ensure a long transaction is not executing against the table being altered. The cluster may block the NBO from acquiring the necessary metadata locks. 
+
+* Nodes cannot be donors for SST during the non-blocking operation. 
+
+* If a node leaves the cluster during the NBO DDL operation the node's data files will be inconsistent. The node can only rejoin the cluster by SST.
+
+* NBO operations can only work with statements that involve one table at a time.
+
+* You cannot run more than one method at a time. 
```

Comment:
this isn't true, you can start an NBO and a TOI transaction at the same time for example, and the cluster will handle it.

Reply:
Removed.